resources/home/dnanexus/make_workbook.py

```
@@ -358,26 +365,23 @@ def person_data(self):
             None
         Outputs:
             None, adds content to openpxyl workbook
-        '''
+        """
         # Get version of HPO to use for terms
         self.get_hpo_obo()
         pb_relate = lambda x: x["additionalInformation"]["relation_to_proband"]
```

Ruff (0.16.0)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `pb_relate` as a `def`

(E731)